src/Sentry/SentryOptions.cs

```diff
@@ -1008,6 +1008,11 @@ public bool JsonPreserveReferences
     /// </summary>
     internal bool InitNativeSdks { get; set; } = true;
 
+    /// <summary>
+    /// Idling transaction timeout after which transaction will be finished automatically.
+    /// </summary>
+    public TimeSpan? IdleTimeout { get; set; }
```

Comment:
```suggestion
    internal TimeSpan? IdleTimeout { get; set; }
```
I think we can keep this internal for now. Once we have an integration that actually uses this we can make it public.